bicep/modules/container-apps/webapp-frontend-service.bicep

```diff
@@ -67,18 +65,13 @@ resource frontendWebAppService 'Microsoft.App/containerApps@2022-06-01-preview'
           name: 'appinsights-key'
           value: appInsightsInstrumentationKey
         }
-        {
-          name: containerRegistryPasswordRefName
-          value: containerRegistryPassword
-        }
       ]
-      registries: [
+      registries: !empty(containerRegistryName) ? [
```

Comment:
@pankajagrawal16 How this will work with option 2 (Import pre-built public images to your private Azure Container Registry") the only option to authenticate with ACR is via managed identity now, username and password will not work, right?

Reply:
Not sure if I understand, Post import, the flow is basically same of authentication via managed identity. Do I miss something here ?

Reply:
Should be fine, I thought in option 2 we are passing the ACR password